dissect/target/plugin.py

```diff
@@ -564,34 +585,34 @@ def child_plugins() -> Iterator[PluginDescriptor]:
     yield from _special_plugins("_child")
 
 
-def lookup(func_name: str, osfilter: str = None) -> Iterator[PluginDescriptor]:
+def lookup(index_name: str, osfilter: str = None) -> Iterator[PluginDescriptor]:
     """Lookup a plugin descriptor by function name.
 
     Args:
-        func_name (str): Function name to lookup.
+        index_name (str): Function name to lookup.
         osfilter (str): OS path the plugin should be from.
     """
-    yield from get_plugins_by_func_name(func_name, osfilter=osfilter)
-    yield from get_plugins_by_namespace(func_name, osfilter=osfilter)
+    yield from get_plugins_by_index_name(index_name, osfilter=osfilter)
+    yield from get_plugins_by_namespace(index_name, osfilter=osfilter)
 
 
-def get_plugins_by_func_name(func_name: str, osfilter: str = None) -> Iterator[PluginDescriptor]:
+def get_plugins_by_index_name(index_name: str, osfilter: str = None) -> Iterator[PluginDescriptor]:
     """Get a plugin descriptor by function name.
 
     Args:
-        func_name (str): Function name to lookup.
+        index_name (str): Function name to lookup.
```

Comment:
Does this mean this now has the same form (e.g. `"namespace.plugin"`) as the keys from `index` returned by `plugin_function_index()`?

Reply:
No, those namespaces are limited to the __namespace__ property, so only one level deep. Or maybe I misunderstand the question?

Reply:
Then wouldn't `func_name`/`get_plugins_by_func_name()` not be more descriptive / less confusing? Now it gives the idea it has something to do with `plugin_function_index()`

Reply:
I think those were changes requested to improve consistency (which it does), but you're right, now I look at it again I think it's a bit confusing on the other hand. Those are really separate functions and I don't see a way to reuse them easily. So better not associate them with the` plugin_function_index()`